scss/_variables.scss

```diff
@@ -38,36 +38,151 @@ $roboto-light: Roboto-Light, $emojifonts, 'Helvetica Neue', Arial, Helvetica, No
 // New colors
 
 html {
-  // '// -' means that it isn't added to theming.js yet and it still needs to be added there
-  --color-signal-blue: #2090ea;
-  --color-core-red: #f44336;
-
-  --color-white: #ffffff;
-  --color-white-07: rgba(#ffffff, 0.7);
-  --color-light-90: #070c14; // some kind of font color?
-  --color-light-35: #a4a6a9;
-  --color-light-60: #62656a;
-  --color-blue: #2090ea;
-  --color-grey: #616161;
-
-  --color-conversation-grey: #505050;
-
-  
-  // Already renamed
-  --outgoing-message-padlock: #4caf50; // TODO: remove this somehow, because it's too specific -> only affects outgoing
-  --context-menu-border: #efefef;
-  --context-menu-bg: #f9fafa;
-  --message-buttons: #8b8e91;
-  --message-buttons-hover: #070c14;
-  --bg-color: #fff;
-  --scrollbar-thumb: #666666; // -
-  --scrollbar-thumb-hover: #606060; // -
+  // clr stands for color to save space 
+  --clr-avatar-label: #fff;
+  --clr-bp3-dialog-header-bg: '#f5f5f5';
+  --clr-bp3-dialog-bg: #ececec;
+  --clr-bp3-dialog-card-bg: #9e9e9e;
+  --clr-bp3-heading: black;
+  --clr-bp3-button-text: '#111111';
+  --clr-bp3-button-bg: '#fff';
+  --clr-broken-media-bg: #070c14;
+  --clr-broken-media-bg-text: #fff;
+  --clr-context-menu-bg: #f9fafa;
+  --clr-context-menu-border: #efefef;
+  --clr-context-menu-text: #070c14;
+  --clr-context-menu-selected: #fff;
+  --clr-context-menu-selected-bg: #a4a6a9;
+  --clr-conversation-list-unread-count-bg: #2090ea;
+  --clr-conversation-list-unread-count-label: #ffffff;
+  --clr-contact-list-item: #62656a;
+  --clr-composer-btn: #616161;
+  --clr-error: #f44336;
+  --clr-global-a: #2090ea;
+  --clr-global-bg: #fff;
+  --clr-global-text: #070c14;
+  --clr-map-overlay-bg: #fff;
+  --clr-message-status-icon: #a4a6a9;
+  --clr-message-status-icon-sending: #62656a;
+  --clr-message-buttons: #8b8e91;
+  --clr-message-buttons-hover: #070c14;
+  --clr-message-padlock-outgoing: #4caf50;
+  --clr-message-padlock-incomming: #a4a6a9;
+  --clr-message-metadata-image-no-caption: #fff;
+  --clr-message-text: #070c14;
+  --clr-message-text-underline: #070c14;
+  --clr-message-generic-attachment-icon-extension: #070c14;
+  --clr-message-generic-attachment-filename: #070c14;
+  --clr-message-generic-attachment-filesize: #070c14;
+  --clr-message-attachment-container-bg: #ffffff;
+  --clr-message-attachment-filename-incomming: #ffffff;
+  --clr-message-attachment-filesize-incomming: #ffffff;
```

Comment:
For me it looks like --clr variables are the real base and they just get pulled in into the --color variables which allow a more specific adjustment. If this is the case, i would prefer to have only one --clr variable for every value and give it a proper name. For example there should be no need to have message-text and message-text-underline variable, that's way to verbose for base colors.

Reply:
$colors you mean? we used this to resolve and group them centraly, we might resolve them again (put the `var(--clr-component-name)` directly in the components).
> For example there should be no need to have message-text and message-text-underline variable
I think one might want to change the link color independently from the message text color (telegram has a different color for links)